prov/efa/src/efa_mr.c

```diff
@@ -176,32 +205,121 @@ struct fi_ops_mr efa_domain_mr_cache_ops = {
 	.regattr = efa_mr_cache_regattr,
 };
 
+static int efa_mr_release(struct efa_mr *mr)
+{
+	struct efa_domain *efa_domain;
+	int ret = 0;
+	int err;
+
+	efa_domain = mr->domain;
+	err = -ibv_dereg_mr(mr->mr);
+	if (!err) {
+		EFA_WARN(FI_LOG_MR,
+			"Unable to close MR\n");
```

Comment:
This warning might be a bit confused. Can we say Unable to deregister memory registration?